package.json

```diff
@@ -16,7 +16,10 @@
     "jest": "^29.7.0",
     "knex": "^3.1.0",
     "mocha": "^10.7.3",
-    "pg": "^8.12.0",
+    "pg": "^8.13.0",
+    "pg-hstore": "^2.3.4",
```

Comment:
What is this used for?

Reply:
Oh it's a Postgres client but since I'm using sequelize I do not need it again so I'll remove it